shared/src/packages/users/libs/validation-schemas/enums/user-validation-message.enum.ts

```diff
@@ -0,0 +1,11 @@
+const UserValidationMessage = {
+  FIELD_IS_REQUIRED: 'This field is mandatory',
+  EMAIL_NOT_VALID: 'This email does not seem valid',
+  PASSWORD_NOT_VALID: 'Must be 6+ characters, at least 1 letter and 1 number',
+  NAME_NOT_VALID: 'Must be 1 - 40 characters, start with a letter, no digits',
+  PHONE_NOT_VALID: 'Must be 8-17 digits, start with +',
```

Comment:
```suggestion
  NAME_NOT_VALID: 'Must be 1 - 40 characters, start with a letter, no digits',
  PHONE_NOT_VALID: 'Must be 8 - 17 digits, start with +',
```
just to be consistent

Reply:
fixed